02-ca-certificates.md

```diff
@@ -13,7 +19,7 @@ Now that you have the [prerequisites](./01-prerequisites.md) met, follow the ste
    Create the certificate for Azure Application Gateway with a common name of `bicycle.contoso.com`.
```

Comment:
I'm struggling to come up with a good way to rephrase this. Maybe something like

Create the certificate for Azure Application Gateway with a common name for your domain. For example, `bicycle.contoso.com`."

Reply:
Could even drop the common name part out:

```suggestion
   Create the certificate that will be presented to web clients by Azure Application Gateway for your domain.
```

What do ya think?

Reply:
👍 I like it.